flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -1970,4 +2041,28 @@ private void areNotificationsEnabled(Result result) {
     NotificationManagerCompat notificationManager = getNotificationManager(applicationContext);
     result.success(notificationManager.areNotificationsEnabled());
   }
+
+  private void setCanScheduleExactNotifications(Result result) {
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S) {
```

Comment:
Thanks for adding this in but what about swapping the if and else logic around? In other words, the if statement would check for the condition `Build.VERSION.SDK_INT >= Build.VERSION_CODES.S`. This would be more consistent with how Android adds API _from_ a particular version of Android so expressing the if statement this way would be consistent with that and how it's been done elsewhere in the plugin like the code you wrote for `checkCanScheduleExactAlarms()` above